ansible_bender/builders/buildah_builder.py

```diff
@@ -164,6 +164,7 @@ def buildah(command, args_and_opts, print_output=False, debug=False, log_stderr=
     cmd = ["buildah"]
     # if debug:
     #     cmd += ["--debug"]
+    cmd += ["--quiet"]
```

Comment:
that's pretty bold to change every buildah command

I'd probably do this only for pull:

https://github.com/ansible-community/ansible-bender/pull/195/files#diff-e820b83f0aadd2f504a891e621c3cfddR41-R44